### DISCUSSION TITLE
💡 Help me to see a character count in text fields

### DISCUSSION THREAD

**Discussion by awaltos** [tinacms/tinacms#4536] [Ideas]
I'd love to see a live character count whilst editing text fields. It would be good if I could enable this as an option in the schema, so it shows only when needed. For example, fields that restrict input length would really benefit having this underneath the field so content editors know how close they are to the limits

![image](https://github.com/tinacms/tinacms/assets/87786621/4f238caa-737f-42bc-b4ee-eb41d9751f3c)
**Figure: I have entered too many chars here. But I don't know how many 😢**

cc: @samsmithssw

**Comment by scottgallant** [comment]
Hi @andrewwaltosssw you should be able to achieve this with a [Custom Field](https://tina.io/docs/extending-tina/custom-field-components/). Would that suffice?

**Reply by awaltos** [reply]
Thanks for the tip! I was able to create a custom field and add that count just below and it is working fine.

I wanted the custom field to also include the label for the input (so I could have the live count next to it) but I wasn't able to hide the default label. If I remove the `label` prop from the field or set it to an empty string then "seo.title" is rendered above my custom field:
![image](https://github.com/tinacms/tinacms/assets/87786621/efad92fe-d3f4-480d-885a-bd4e4016144e)
**Figure: The default label that I'd like to hide**

Is there a way to hide the default label?
